pages/en/about/security-reporting.md

```diff
@@ -72,3 +72,9 @@ Security notifications will be distributed via the following methods.
 If you have suggestions on how this process could be improved please submit a
 [pull request](https://github.com/nodejs/nodejs.org) or
 [file an issue](https://github.com/nodejs/security-wg/issues/new) to discuss.
+
+## OpenSSF Best Practices
+
+[![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/29/badge)](https://bestpractices.coreinfrastructure.org/projects/29)
```

Comment:
Move after the parahraph?

Reply:
Badges usually go before related prose

Reply:
I get that, but the issue is this excessive padding 😅

Reply:
How do i see the padding?

Reply:
https://nodejs-org-git-openssf-badge-openjs.vercel.app/en/about/security-reporting#openssf-best-practices

here's a mockup of what it would look like below

| current | propose |
| - | - | 
| ![image](https://github.com/nodejs/nodejs.org/assets/298435/56646efe-a121-4f83-8a15-766986652017) | ![image](https://github.com/nodejs/nodejs.org/assets/298435/6dbf8305-a312-41e5-b204-b2085671b749) |

Reply:
oh, well yeah that padding is very subpar - there's no way to remove the padding for that image?

Reply:
https://github.com/nodejs/nodejs.org/pull/6057/commits/1c58f58af5013a472788824ee8d981c2f9f6878e is the smallest possible change I could find to:

- remove the extra padding from global css
- remove the worst of the hover/focus artifacts

![image](https://github.com/nodejs/nodejs.org/assets/298435/6db911b2-8e16-444c-9a0b-c2d615b36078)

![image](https://github.com/nodejs/nodejs.org/assets/298435/5468e91c-745e-4149-af91-a415aee6e2db)

this code will be revisited as part of the redesign. I think it's good enough for now

Reply:
Looks good to me!